<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addF</class>
 <widget class="QDialog" name="addF">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_nome">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_criar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>80</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Criar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_equipe">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Equipe</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_nome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_equipe">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbx_numFunc">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Funcionário 01</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Funcionário 02</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Funcionário 03</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Funcionário 04</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Funcionário 05</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Funcionário 06</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Funcionário 07</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Funcionário 08</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Funcionário 09</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Funcionário 10</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_remover">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Remover</string>
   </property>
  </widget>
  <widget class="QLabel" name="diretorio">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Ano">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Escala">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
